.env.example

```diff
@@ -1,15 +1,23 @@
+NODE_ENV=development
+
 DATABASE_CLIENT=mysql2
 DATABASE_USERNAME=vim
 DATABASE_PASSWORD=super-secret
 DATABASE_PORT=3306
 DATABASE_SSL=false
 
 WEB_DATABASE_NAME=vim
-CMS_DATABASE_NAME=vim_cms
+NEXT_PRIVATE_STANDALONE=true
+NEXTAUTH_SECRET=testsecret
 
+CMS_DATABASE_NAME=vim_cms
+CMS_URL="https://preview.vim.org/cms"
```

Comment:
Can we delete this?

Reply:
True we can, since local development does not set the CMS_URL